### DISCUSSION TITLE
Power mode

### DISCUSSION THREAD

**Discussion by wrccclight** [seerge/g-helper#2643] [Ideas]
The current software is based on balanced mode. Can high-performance mode be added in the future, or can power mode be freely chosen? In my computer system, I can choose high-performance mode while using gelper to limit power consumption, and the frequency can reach higher levels

**Comment by seerge** [comment]
@wrccclight hello,

If you speak about windows power plans then you can assign any plan you want via config 
https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-windows-power-plans-with-each-mode

Even tho it’s not much sense in that, as in modern windows power plans are practically substituted with power modes that can be changed via app easily

**Comment by wrccclight** [comment]
Thank you for your help. After the change, the frequency can be maintained above 4000GHz and the temperature is similar to the equilibrium mode
